tests/io/FileTest.py

```diff
@@ -12,19 +15,17 @@ def get_path_components(filepath):
     return os.path.normpath(filepath).split(os.sep)
 
 
-class FileFactoryTest(unittest.TestCase):
+class FileTest(unittest.TestCase):
 
     def setUp(self):
-        file_factory_test_dir = os.path.join(os.path.split(__file__)[0],
-                                             'FileFactoryTestFiles')
-
+        file_factory_test_dir = TEST_FILE_DIR
         self.test_file = os.path.join(file_factory_test_dir, 'test1.txt')
         self.other_test_file = os.path.join(file_factory_test_dir, 'test2.txt')
-        self.uut = FileFactory(self.test_file)
-        self.other_file_factory = FileFactory(self.other_test_file)
+        self.uut = File(self.test_file)
+        self.other_file_factory = File(self.other_test_file)
```

Comment:
`other_file_factory` --> `other_file`?

Reply:
The intention is to reduce the amount of change occurring in the tests, so that any functional changes are more obvious.  Any change to tests is a possible breakage.

The focus is on our docs and code which people read all the time.

The name of a variable in a test can be done as a newcomer issue later.